src/core/lib/layer.js

```diff
@@ -139,6 +139,16 @@ export default class Layer {
   // App can destroy WebGL resources here
   finalizeState() {}
 
+  // Update attribute transition
+  updateTransition() {
+    const {model, attributeManager} = this.state;
```

Comment:
Use `getModels()`.

Reply:
`updateAttributes` does not use `getModels` either. We do not automatically update attributes for multi-model layers.

Reply:
mainly want to avoid accessing state.model directly. Use the first model from `getModels`, and assert if there are more, or add a `getModel` method (maybe already there?).

Reply:
will do in another PR as it modifies existing functionality.